{"name": "Проверка: при создании нового заказа счётчик \"Выполнено за сегодня\" увеличивается", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF76631EEA2+31554]\n\t(No symbol) [0x00007FF766297ED9]\n\t(No symbol) [0x00007FF76615872A]\n\t(No symbol) [0x00007FF7661A8434]\n\t(No symbol) [0x00007FF7661A853C]\n\t(No symbol) [0x00007FF7661EF6A7]\n\t(No symbol) [0x00007FF7661CD06F]\n\t(No symbol) [0x00007FF7661EC977]\n\t(No symbol) [0x00007FF7661CCDD3]\n\t(No symbol) [0x00007FF76619A33B]\n\t(No symbol) [0x00007FF76619AED1]\n\tGetHandleVerifier [0x00007FF766628B1D+3217341]\n\tGetHandleVerifier [0x00007FF766675AE3+3532675]\n\tGetHandleVerifier [0x00007FF76666B0E0+3489152]\n\tGetHandleVerifier [0x00007FF7663CE776+750614]\n\t(No symbol) [0x00007FF7662A375F]\n\t(No symbol) [0x00007FF76629EB14]\n\t(No symbol) [0x00007FF76629ECA2]\n\t(No symbol) [0x00007FF76628E16F]\n\tBaseThreadInitThunk [0x00007FFF5AFA257D+29]\n\tRtlUserThreadStart [0x00007FFF5C6EAF28+40]", "trace": "self = <tests.test_lent_of_orders.TestLentOrders object at 0x0000025B868527B0>, driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"63b55254fef3056d7e2a06aa15d8ee36\")>, set_token = None\n\n    @allure.title('Проверка: при создании нового заказа счётчик \"Выполнено за сегодня\" увеличивается') #-+\n    def test_create_new_order_counter_today_rise(self, driver, set_token):\n        main_page = MainPage(driver)\n        food_page = FoodPage(driver)\n        main_page.click_lent_of_orders_fox()\n>       order_1 = food_page.get_order_counter_for_today()\n\ntests\\test_lent_of_orders.py:51: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_objects\\food_page.py:27: in get_order_counter_for_today\n    self.find_element_waiting(FoodLocators.ORDER_COUNTER_TODAY)\npage_objects\\base_page.py:16: in find_element_waiting\n    self.wait_visibility_of_element(locator)\npage_objects\\base_page.py:30: in wait_visibility_of_element\n    return WebDriverWait(self.driver, 20).until(expected_conditions.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"63b55254fef3056d7e2a06aa15d8ee36\")>, method = <function visibility_of_element_located.<locals>._predicate at 0x0000025B86876020>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF76631EEA2+31554]\nE       \t(No symbol) [0x00007FF766297ED9]\nE       \t(No symbol) [0x00007FF76615872A]\nE       \t(No symbol) [0x00007FF7661A8434]\nE       \t(No symbol) [0x00007FF7661A853C]\nE       \t(No symbol) [0x00007FF7661EF6A7]\nE       \t(No symbol) [0x00007FF7661CD06F]\nE       \t(No symbol) [0x00007FF7661EC977]\nE       \t(No symbol) [0x00007FF7661CCDD3]\nE       \t(No symbol) [0x00007FF76619A33B]\nE       \t(No symbol) [0x00007FF76619AED1]\nE       \tGetHandleVerifier [0x00007FF766628B1D+3217341]\nE       \tGetHandleVerifier [0x00007FF766675AE3+3532675]\nE       \tGetHandleVerifier [0x00007FF76666B0E0+3489152]\nE       \tGetHandleVerifier [0x00007FF7663CE776+750614]\nE       \t(No symbol) [0x00007FF7662A375F]\nE       \t(No symbol) [0x00007FF76629EB14]\nE       \t(No symbol) [0x00007FF76629ECA2]\nE       \t(No symbol) [0x00007FF76628E16F]\nE       \tBaseThreadInitThunk [0x00007FFF5AFA257D+29]\nE       \tRtlUserThreadStart [0x00007FFF5C6EAF28+40]\n\n..\\..\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Клик по ленте заказов для Мозилы", "status": "passed", "start": 1721156747133, "stop": 1721156747433}, {"name": "Количество заказов за сегодня", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF76631EEA2+31554]\n\t(No symbol) [0x00007FF766297ED9]\n\t(No symbol) [0x00007FF76615872A]\n\t(No symbol) [0x00007FF7661A8434]\n\t(No symbol) [0x00007FF7661A853C]\n\t(No symbol) [0x00007FF7661EF6A7]\n\t(No symbol) [0x00007FF7661CD06F]\n\t(No symbol) [0x00007FF7661EC977]\n\t(No symbol) [0x00007FF7661CCDD3]\n\t(No symbol) [0x00007FF76619A33B]\n\t(No symbol) [0x00007FF76619AED1]\n\tGetHandleVerifier [0x00007FF766628B1D+3217341]\n\tGetHandleVerifier [0x00007FF766675AE3+3532675]\n\tGetHandleVerifier [0x00007FF76666B0E0+3489152]\n\tGetHandleVerifier [0x00007FF7663CE776+750614]\n\t(No symbol) [0x00007FF7662A375F]\n\t(No symbol) [0x00007FF76629EB14]\n\t(No symbol) [0x00007FF76629ECA2]\n\t(No symbol) [0x00007FF76628E16F]\n\tBaseThreadInitThunk [0x00007FFF5AFA257D+29]\n\tRtlUserThreadStart [0x00007FFF5C6EAF28+40]\n\n", "trace": "  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\food_page.py\", line 27, in get_order_counter_for_today\n    self.find_element_waiting(FoodLocators.ORDER_COUNTER_TODAY)\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\base_page.py\", line 16, in find_element_waiting\n    self.wait_visibility_of_element(locator)\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\base_page.py\", line 30, in wait_visibility_of_element\n    return WebDriverWait(self.driver, 20).until(expected_conditions.visibility_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Найти элемент на странице", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF76631EEA2+31554]\n\t(No symbol) [0x00007FF766297ED9]\n\t(No symbol) [0x00007FF76615872A]\n\t(No symbol) [0x00007FF7661A8434]\n\t(No symbol) [0x00007FF7661A853C]\n\t(No symbol) [0x00007FF7661EF6A7]\n\t(No symbol) [0x00007FF7661CD06F]\n\t(No symbol) [0x00007FF7661EC977]\n\t(No symbol) [0x00007FF7661CCDD3]\n\t(No symbol) [0x00007FF76619A33B]\n\t(No symbol) [0x00007FF76619AED1]\n\tGetHandleVerifier [0x00007FF766628B1D+3217341]\n\tGetHandleVerifier [0x00007FF766675AE3+3532675]\n\tGetHandleVerifier [0x00007FF76666B0E0+3489152]\n\tGetHandleVerifier [0x00007FF7663CE776+750614]\n\t(No symbol) [0x00007FF7662A375F]\n\t(No symbol) [0x00007FF76629EB14]\n\t(No symbol) [0x00007FF76629ECA2]\n\t(No symbol) [0x00007FF76628E16F]\n\tBaseThreadInitThunk [0x00007FFF5AFA257D+29]\n\tRtlUserThreadStart [0x00007FFF5C6EAF28+40]\n\n", "trace": "  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\base_page.py\", line 16, in find_element_waiting\n    self.wait_visibility_of_element(locator)\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\base_page.py\", line 30, in wait_visibility_of_element\n    return WebDriverWait(self.driver, 20).until(expected_conditions.visibility_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Прогрузка элемента", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF76631EEA2+31554]\n\t(No symbol) [0x00007FF766297ED9]\n\t(No symbol) [0x00007FF76615872A]\n\t(No symbol) [0x00007FF7661A8434]\n\t(No symbol) [0x00007FF7661A853C]\n\t(No symbol) [0x00007FF7661EF6A7]\n\t(No symbol) [0x00007FF7661CD06F]\n\t(No symbol) [0x00007FF7661EC977]\n\t(No symbol) [0x00007FF7661CCDD3]\n\t(No symbol) [0x00007FF76619A33B]\n\t(No symbol) [0x00007FF76619AED1]\n\tGetHandleVerifier [0x00007FF766628B1D+3217341]\n\tGetHandleVerifier [0x00007FF766675AE3+3532675]\n\tGetHandleVerifier [0x00007FF76666B0E0+3489152]\n\tGetHandleVerifier [0x00007FF7663CE776+750614]\n\t(No symbol) [0x00007FF7662A375F]\n\t(No symbol) [0x00007FF76629EB14]\n\t(No symbol) [0x00007FF76629ECA2]\n\t(No symbol) [0x00007FF76628E16F]\n\tBaseThreadInitThunk [0x00007FFF5AFA257D+29]\n\tRtlUserThreadStart [0x00007FFF5C6EAF28+40]\n\n", "trace": "  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\PycharmProjects\\Diplom_3\\page_objects\\base_page.py\", line 30, in wait_visibility_of_element\n    return WebDriverWait(self.driver, 20).until(expected_conditions.visibility_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Anastasia\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python312\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Выполнено за сегодня:\"]/following-sibling::p')"}], "start": 1721156747433, "stop": 1721156767796}], "parameters": [{"name": "locator", "value": "('xpath', '//p[text()=\"Выполнено за сегодня:\"]/following-sibling::p')"}], "start": 1721156747433, "stop": 1721156767796}], "start": 1721156747433, "stop": 1721156767796}], "parameters": [{"name": "driver", "value": "'chrome'"}], "start": 1721156747133, "stop": 1721156767798, "uuid": "74467ca7-cbe5-4530-be30-3d865e9e595b", "historyId": "3d45ec768fb522f00e1490ee8714426b", "testCaseId": "36ac1f4ec2ee9b401aa2dfdfb3e37393", "fullName": "tests.test_lent_of_orders.TestLentOrders#test_create_new_order_counter_today_rise", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_lent_of_orders"}, {"name": "subSuite", "value": "TestLentOrders"}, {"name": "host", "value": "Pro_Magnus_M"}, {"name": "thread", "value": "18484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_lent_of_orders"}]}